harness/determined/cli/shell.py

```diff
@@ -90,6 +90,14 @@ def show_ssh_command(args: Namespace) -> None:
     )
 
 
+@authentication.required
```

Comment:
I don't think this is necessary, since the next function you call is already wrapped (right?)

I think it might actually cause additional network traffic, so I think it's not harmless.